proto/p4/config/p4info.proto

```diff
@@ -197,3 +198,19 @@ message DirectMeter {
   // the id of the table to which the meter is attached
   uint32 direct_table_id = 3;
 }
+
+// packet-in & packet-out metadata (consistent with p4runtime.proto)
+message ControllerPacketMetadata {
```

Comment:
For my info, in p4c, the P4 runtime serializer is expected to add this to the P4Info based on header definitions?